README.md

```diff
@@ -180,7 +180,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [X](#x) | [Y](#y) | [Z](#z) |
 
 - ### **A**
-
```

Comment:
You have remove a new line before you added your name. Please add this new line. Other than this, everything is ok.